rewrite-gradle/src/test/java/org/openrewrite/gradle/AddGradleWrapperTest.java

```diff
@@ -108,6 +101,67 @@ void addWrapper() {
         );
     }
 
+
+    @Test
+    void addWrapperWithReleaseTag() {
+
+        AtomicReference<String> output = new AtomicReference<>("");
+        rewriteRun(
+          spec ->
+            spec
+              .recipe(
+                //language=yaml
+                new ByteArrayInputStream(
+                  """
+                    ---
+                    type: specs.openrewrite.org/v1beta/recipe
+                    name: org.openrewrite.test.AddGradleWrapper
+                    displayName: Adds a Gradle wrapper
+                    description: Adds the latest RELEASE of the gradle wrapper
+                    recipeList:
+                      - org.openrewrite.gradle.AddGradleWrapper
+                    """.getBytes()
+                ),
+                "org.openrewrite.test.AddGradleWrapper"
+              )
+              .afterRecipe(run -> {
+
+                  validateGradlewFiles(run);
+
+                  var gradleWrapperJar = result(run, Remote.class, "gradle-wrapper.jar");
+                  assertThat(PathUtils.equalIgnoringSeparators(gradleWrapperJar.getSourcePath(), WRAPPER_JAR_LOCATION)).isTrue();
+                  assertThat(gradleWrapperJar.getUri().toString()).endsWith("-bin.zip");
+
+                  var wrapperProperties =  run.getResults().stream()
+                    .map(Result::getAfter)
+                    .filter(Objects::nonNull)
+                    .filter(r -> r.getSourcePath().endsWith("gradle-wrapper.properties"))
+                    .findFirst().orElseThrow();
+
+                  output.set(wrapperProperties.printAll());
+
+              }),
+
+          buildGradle(""),
+
+          dir(
+            "gradle/wrapper",
+            properties(null, output, spec -> spec.path(Paths.get("gradle-wrapper.properties")))
+          ));
```

Comment:
This assertion could be better if we had a mechanism to mock the response. This value is dynamic so I did not find any other better way to verify it other than this

Reply:
So you can do the assertion with the `Assertions.properties` call. And I believe that there is a way to mock the dynamic versions now.

@jkschneider, do you have any insight on that mocking support?

Reply:
Yes, I'll put up a polish commit showing how to do it.

Reply:
@shanman190 I believe that I applied all your feedback in my latest commits :) 

@jkschneider Did you get a chance to set up the example you mentioned?

Reply:
So the `org.openrewrite.properties.Assertions.properties(..)` method handles the directory pathing so that you don't have to. Also not sure of the value of capturing the output instead of just hard coding the assertion and seeing what Jon has in store from a mocking standpoint when he submits his polish commit on top.

Reply:
I hard-coded it to `7.6` for now and the PR is "ready". This test is bound to fail in the future when a new version comes out so we can wait for Jon to share an example for the mocking before we merge :)